backend/.env

```
@@ -14,18 +14,15 @@
 
 # DOMAIN=
 
-# EMAIL_USER=
-# EMAIL_PASS=
+EMAIL_USER=maruthikummari553@gmail.com
+EMAIL_PASS=lhcn duvu ahtw dwvh
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_PASS key should go before the EMAIL_USER key

(UnorderedKey)